addon/components/sl-date-picker.js

```diff
@@ -68,6 +68,13 @@ export default Ember.Component.extend( TooltipEnabled, {
     daysOfWeekDisabled: [],
 
     /**
+     * When true, the datepicker will not be displayed
```

Comment:
well....the input field will display but be disabled and not able to be interacted with in order to select the date selection calendar.  Perhaps tweak this working just a small bit?